bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -515,8 +525,30 @@ func makeCreateAppEndpoint(svc KsService) endpoint.Endpoint {
 		req := request.(CreateRequest)
 		err := svc.CreateApp(ctx, req)
 
-		r := &createResponse{}
+		r := &basicServerResponse{}
+
+		if err != nil {
+			r.Err = err.Error()
+		}
+		return r, nil
+	}
+}
+
+func makeLivenessEndpoint(svc KsService) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(LiveRequest)
+		r := &LiveResponse{}
+		r.Reply = req.Msg + "accepted! Sill alive!"
```

Comment:
Are there no other checks we want to perform here to make sure the entire system is up and running? If the endpoint returns a success code, can the caller then assume that all other services are up.. etc?

Reply:
Currently this one only indicate backend is reachable and have cycles to respond to requests.

We can keep a server status variable to indicate backend status, which should kill the pod if needed so we have auto-start.
Let's create issue for it.